sdk/go/gcp/organizations/getFolder.go

```diff
@@ -10,6 +10,26 @@ import (
 // Use this data source to get information about a Google Cloud Folder.
 // 
 // ```hcl
+// # Get folder by id
```

Comment:
Looks like this is a code snippet embedded inside of the resource's top level comment, not within an example section.